ament_cmake_export_dependencies/cmake/ament_cmake_export_dependencies-extras.cmake.in

```diff
@@ -36,14 +36,17 @@ if(NOT _exported_dependencies STREQUAL "")
 
           get_target_property(_imported_configurations ${_target} IMPORTED_CONFIGURATIONS)
           if(_imported_configurations)
-            list(TRANSFORM _imported_configurations TOUPPER)
```

Comment:
Wouldn't it be sufficient to just replace this line with `string(TOUPPER "${_imported_configurations}" _imported_configurations)`?

Reply:
Yes you are right, it is much simpler, I did not thought to convert the list to string and then apply conversion. Thanks, updated.

BR,
Csaba